bigframes/_config/compute_options.py

```diff
@@ -66,13 +66,19 @@ class ComputeOptions:
             engine to handle. However this comes at the cost of increase cost and latency.
         extra_query_labels (Dict[str, Any], Options):
             Stores additional custom labels for query configuration.
+        sem_ops_confirmation_threshold (int, Options):
+            Guards against unexepcted processing of large amount of rows by semantic operators.
+            If the number of rows exceeds the threshold, the user will be asked to confirm
+            their operations to resume. The default value is 25. Set the value to None
+            to turn off the guard.
     """
 
     maximum_bytes_billed: Optional[int] = None
     enable_multi_query_execution: bool = False
     extra_query_labels: Dict[str, Any] = dataclasses.field(
         default_factory=dict, init=False
     )
+    sem_ops_confirmation_threshold: Optional[int] = 25
```

Comment:
I worry if someone tries this on small data and then tries to automate it, their script will hang waiting for input. Please set to 1 / 0 so that we always show this message and folks will know to turn it off for automated use cases.